Metacity should respond to this too, not just Mutter

--- origsrc/gnome-tweak-tool-3.10.1/gtweak/tweaks/tweak_group_windows.py	2013-09-19 04:32:20.000000000 -0500
+++ src/gnome-tweak-tool-3.10.1/gtweak/tweaks/tweak_group_windows.py	2014-03-26 14:41:54.293060400 -0500
@@ -31,7 +31,6 @@ class ShowWindowButtons(GSettingsSwitchT
                                            name,
                                            "org.gnome.desktop.wm.preferences",
                                            "button-layout",
-                                           loaded=_shell_loaded,
                                            **options)
         self.values=[':close',':minimize,close', ':maximize,close', ':minimize,maximize,close']
         
